### DISCUSSION TITLE
Spotify Inspired Feature Request (Year Wrap)

### DISCUSSION THREAD

**Discussion by saltsoftdrink** [monkeytypegame/monkeytype#6136] [Ideas]
Is it possible for someone, or even the owner of Monkeytype, to create a feature or integrate it into the website that provides an annual wrap-up for users? This could include personalized insights such as:  

1. **Total Typing Hours**: How many hours you typed throughout the year.  
2. **Longest Typing Session**: The duration of your longest uninterrupted typing session.  
3. **Typing Progress**: How much your words per minute (WPM) improved over the year.  
4. **Favorite Typing Settings**: Insights into your most-used preferences, such as session types (e.g., 15 seconds, 10 words) or word lists (e.g., English or English 1k).  
5. **Additional Statistics**: Any other key metrics that highlight your typing journey, like accuracy trends or word categories.  

This annual summary would be a great way for users to reflect on their progress and set new goals!

**Comment by saltsoftdrink** [comment]
@Miodec

**Comment by maxhu08** [comment]
Would be cool

**Comment by unrealapex** [comment]
While having a wrap-up would be really nice to see. It probably won't become a features unless someone is open to work on it. Your [account profile](https://monkeytype.com/account) contains all this information you requested in the form of unexciting charts, graphs, and statistics.

edit: coincidentally, I requested for this feature three years ago: #1783.

**Reply by hexxt-git** [reply]
yup its pretty late
